src/Game.java

```diff
@@ -0,0 +1,35 @@
+public class Game {
+
+    Players playerOne;
+    Players playerTwo;
+    Players scorePlayerOne = new Players();
```

Comment:
This is still the old code. I guess you have forgot to commit 😉

Reply:
Should I wait before reviewing this?